### PR TITLE
Bump version to 1.0.0rc8

### PR DESCRIPTION
**Pull request by JingweiZhang12**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,16 +43,17 @@
...
 ### 🌟 Preview of 1.1.x version
 
-A brand new version of **MMDetection v1.1.0rc3** was released in 7/1/2023:
+A brand new version of **MMDetection v1.1.0rc4** was released in 26/3/2023:
 
 - Unifies interfaces of all components based on [MMEngine](https://github.com/open-mmlab/mmengine) and [MMDet 3.x](https://github.com/open-mmlab/mmdetection/tree/3.x).
 - A standard data protocol defines and unifies the common keys across different datasets.
 - Faster training and testing speed with more strong baselines.
 
+We have supported more LiDAR-based segmentation methods. More new features about 3D perception are in process.
```
Comment (Tai-Wang) [inline README.md:56]: segmentation methods, including xxx, xxx, xxx

Comment (Tai-Wang) [reply README.md:56]: More new features are on the way / in progress

File: README_zh-CN.md [hunk 2]
```diff
@@ -28,16 +28,17 @@
...
 ### 🌟 1.1.x 预览版本
 
-全新的 **v1.1.0rc3** 版本已经在 2023.1.7 发布：
+全新的 **v1.1.0rc4** 版本已经在 2023.3.26 发布：
 
 - 基于 [MMEngine](https://github.com/open-mmlab/mmengine) 和 [MMDet 3.x](https://github.com/open-mmlab/mmdetection/tree/3.x) 统一了各组件接口。
 - 通过一个标准的数据格式定义和统一了不同数据集的通用内容。
 - 实现了更快的训练和测试速度，并提供了更多强大的基准模型。
 
+我们已经支持了更多基于 LiDAR 的 3D 分割算法。更多关于 3D 感知的新特性正在开发中！
```
Comment (Tai-Wang) [inline README_zh-CN.md:41]: 包括哪些

File: docs/en/changelog.md [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+### v1.0.0rc8 (26/3/2023)
+
+#### Improvements
+
+- Add BN in FPN to avoid loss Nan in MVXNet (#2282)
+- Update s3dis_data_utils.py (#2232)
+
+#### Bug Fixes
+
+- Fix pecision error when using mixed precision on CenterPoint (#2341)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:12]: pecision -> precision

File: docs/en/changelog.md [hunk 4]
```diff
@@ -1,5 +1,23 @@
...
+- Replace np.transpose by torch.permute to speed up (@2273)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:13]: \#2273

File: README_zh-CN.md [hunk 5]
```diff
@@ -28,16 +28,17 @@
...
 ### 🌟 1.1.x 预览版本
 
-全新的 **v1.1.0rc3** 版本已经在 2023.1.7 发布：
+全新的 **v1.1.0rc4** 版本已经在 2023.3.26 发布：
 
 - 基于 [MMEngine](https://github.com/open-mmlab/mmengine) 和 [MMDet 3.x](https://github.com/open-mmlab/mmdetection/tree/3.x) 统一了各组件接口。
 - 通过一个标准的数据格式定义和统一了不同数据集的通用内容。
 - 实现了更快的训练和测试速度，并提供了更多强大的基准模型。
 
+我们已经支持了更多基于 LiDAR 的 3D 分割算法, 包括 Cylinder3d, MinkUNet 和 SPVCNN。更多关于 3D 感知的新特性正在开发中！
```
Comment (ZCMax) [inline README_zh-CN.md:41]: #### 亮点

我们已经支持了更多基于 LiDAR 的 3D 分割算法, 包括 **Cylinder3d**, **MinkUNet** 和 **SPVCNN**。更多关于 3D 感知的新特性正在开发中！

**Pull request closed without merging**